src/main/java/com/cburch/logisim/fpga/gui/FPGACommander.java

```diff
@@ -54,21 +54,11 @@
 import java.awt.Dimension;
 import java.awt.GridBagConstraints;
 import java.awt.GridBagLayout;
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
-import java.awt.event.WindowEvent;
-import java.awt.event.WindowListener;
+import java.awt.event.*;
```

Comment:
Personally, I prefer to explicitly import only the used classes.

Reply:
Fixed

Reply:
Thanks!